aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/input/InputStepTest.java

```diff
@@ -52,11 +54,14 @@ public void parameter() throws Exception {
         InputStepExecution is = a.getExecution("Icecream");
         assertEquals("Do you want chocolate?", is.getInput().getMessage());
         assertEquals(1, is.getInput().getParameters().size());
+        assertEquals("alice", is.getInput().getSubmitter());
```

Comment:
I think I am just comparing "Alice" with the submitter parameter defined in the input set definition above:
 assertEquals("alice", is.getInput().getSubmitter()); does not look like failing